### PR TITLE
feat: avatar bubble size and change behavior

### PR DESCRIPTION
**Pull request by imGok**
demoavatar

### CODE REVIEW

File: lib/pages/chat/chat_event_list.dart [hunk 1]
```diff
@@ -102,7 +102,10 @@ class ChatEventList extends StatelessWidget {
 
           // The message at this index:
           final event = controller.timeline!.events[i - 1];
-
+          final previousEvent = i > 2 ? controller.timeline!.events[i - 2] : null;
```
Comment (hoangdat) [inline lib/pages/chat/chat_event_list.dart:105]: what is `2`? for the same convention in the future, dont use magic number, please use constant to make the code readable

Comment (imGok) [reply lib/pages/chat/chat_event_list.dart:105]: events are displayed in reverse and current event is index - 1, previous is index - 2

**Review by hoangdat — APPROVED** [review]
please rebase

**Comment by imGok** [comment]
@sherlockvn you can see reply demo in description; this is reply 

demovatara

File: lib/pages/chat/events/message.dart [hunk 2]
```diff
@@ -106,6 +100,20 @@ class Message extends StatelessWidget {
...
         }.contains(event.messageType);
 
+        final isPreviousEventMessage = {
+              EventTypes.Message,
+              EventTypes.Sticker,
+              EventTypes.Encrypted,
+              EventTypes.Redaction,
+            }.contains(previousEvent?.type);
+
+        final sameSender = previousEvent == null ||
```
Comment (sherlockvn) [inline lib/pages/chat/events/message.dart:110]: Can you break it down, this is hard to understand @@

Comment (hoangdat) [reply lib/pages/chat/events/message.dart:110]: deserve a method

Comment (imGok) [reply lib/pages/chat/events/message.dart]: @hoangdat @sherlockvn check [988cf68](https://github.com/linagora/twake-on-matrix/pull/302/commits/988cf68f9818cb0aa3a665d6031d0273e42749df) please

File: lib/pages/chat/events/message.dart [hunk 3]
```diff
@@ -515,6 +511,23 @@ class Message extends StatelessWidget {
...
+    final isPreviousEventMessage = {
+      EventTypes.Message,
+      EventTypes.Sticker,
+      EventTypes.Encrypted,
+      EventTypes.Redaction,
+    }.contains(previousEvent.type);
+
+    return isPreviousEventMessage
+      ? previousEvent.senderId != event.senderId
+      : !isPreviousEventMessage;
```
Comment (hoangdat) [inline lib/pages/chat/events/message.dart:529]: hard to understand @@

Comment (imGok) [reply lib/pages/chat/events/message.dart]: I broke down the ternary and added comments

**Comment by hoangdat** [comment]
Please rebase

**Review by hoangdat — APPROVED** [review]

**Pull request merged**